packages/wonder-blocks-labeled-field/src/components/labeled-field.tsx

```diff
@@ -184,27 +225,40 @@ export default function LabeledField(props: Props) {
 
 const styles = StyleSheet.create({
     label: {
-        color: color.offBlack,
+        color: semanticColor.text.primary,
     },
     lightLabel: {
-        color: color.white,
+        color: semanticColor.text.inverse,
     },
     description: {
-        color: color.offBlack64,
+        color: semanticColor.text.secondary,
     },
     lightDescription: {
         color: color.white64,
```

Comment:
We don't have semantic color tokens yet for colors used on dark backgrounds so the original color tokens are still used for the light styling (except for the label since we were able to use the text inverse token)!